fortinet_fortigate/plugin.spec.yaml

```
@@ -612,18 +612,43 @@ connection:
     title: Hostname
     description: Hostname or IP of your FortiGate server e.g. myfortigate.internal, 192.168.10.1, 192.168.10.1:8000
     required: true
+    example: fortigate.rapid7.com
   api_key:
     type: credential_secret_key
     title: API Key
     description: API key
     required: true
+    example: 2Fty5834tFpBdidePJnt9075MMdkUb
   ssl_verify:
     type: boolean
     title: SSL Verify
     description: SSL verify
     required: true
+    example: false
 
 actions:
+  check_if_ip_is_in_address_groups:
+    title: Check If IP is in Address Group
```

```suggestion
    title: Check Address Group for IP Address
```
Just a bit more concise of a title